principal_travel/templates/travel/principal_list.html

```diff
@@ -1,26 +1,21 @@
 {% extends 'base.html' %}
 {% block content %}
-<h4>Principals</h4>
+<h3>Principals   <a href="{% url 'principal_new' %}" class="btn btn-success add-button">Add</a></h3>
 <div>
-  <table class="table table-striped">
+  <table class="table table-hover">
     <thead class="thead-default">
       <tr>
-        <th scope='col'> </th>
         <th scope='col'> Name </th>
         <th scope='col'> Title </th>
         <th scope='col'> Office </th>
       </tr>
     </thead>
-    <tbody>
+    <tbody onclick="getElementById('edit-1').click()" style="cursor: pointer">
```

Comment:
The clickable rows look GREA! But is there a reason to use the `edit-1` id? It doesn't seem very semantic.